regulations/static/regulations/js/source/views/main/child-view.js

```diff
@@ -77,8 +77,9 @@ var ChildView = Backbone.View.extend({
     render: function() {
         this.updateWayfinding();
         this.loadImages();
-        HeaderEvents.trigger('section:open', this.id);
-        DrawerEvents.trigger('section:open', this.id);
+        // TODO: What is this for?
+        // HeaderEvents.trigger('section:open', this.id);
+        // DrawerEvents.trigger('section:open', this.id);
```

Comment:
`DrawerEvents` is needed, lest navigating through an internal citation results in the wrong section being highlighted:

<img width="591" alt="screen shot 2016-04-14 at 1 08 35 pm" src="https://cloud.githubusercontent.com/assets/326918/14536849/6c191b60-0242-11e6-98ab-f7e00320aacf.png">